steward/src/commands/schedule_corks/aave_cellar/shutdown.rs

```diff
@@ -0,0 +1,78 @@
+use crate::{
+    application::APP,
+    cellars::{self},
+    config,
+    prelude::*,
+    somm_send,
+};
+use abscissa_core::{clap::Parser, Application, Command, Runnable};
+use deep_space::{Coin, Contact};
+use ethers::abi::AbiEncode;
+use somm_proto::cork::Cork;
+use std::time::Duration;
+use steward_abi::aave_v2_stablecoin::*;
+const MESSAGE_TIMEOUT: Duration = Duration::from_secs(10);
+const CHAIN_PREFIX: &str = "somm";
+
+/// Shutdown subcommand
+#[derive(Command, Debug, Parser)]
+#[clap(
+    long_about = "DESCRIPTION \n\n Shutdown target Cellar.\n This command schedules the shutdown of a Cellar. This is a validator only \n command and can only be run by validators. It Schedules shutdown based on the height specified by \n the validators. Therefore, it'll execute the function when the chain reaches that height."
+)]
+pub struct ShutdownCmd {
+    /// Target contract for scheduled cork.
+    #[clap(short, long)]
+    contract: String,
+
+    /// Block height to schedule cork.
+    #[clap(short = 'b', long)]
+    height: u64,
+}
+
+impl Runnable for ShutdownCmd {
+    fn run(&self) {
+        let config = APP.config();
+
+        abscissa_tokio::run_with_actix(&APP, async {
+            let call = SetShutdownCall {
+                shutdown: true,
+                exit_position: true,
+            };
+            let encoded_call = AaveV2StablecoinCellarCalls::SetShutdown(call).encode();
+
+            cellars::validate_cellar_id(self.contract.as_str()).unwrap_or_else(|err| {
+                status_err!("Can't validate cellar ID: {}", err);
```

Comment:
We should probably change this message to something like "can't validate contract address format" since the command isn't really calling them cellar IDs.